public/v2/apps/etesync.json

```diff
@@ -0,0 +1,54 @@
+{
+    "captainVersion": "2",
+    "documentation": "Read the documentation @ https://github.com/GRBurst/docker-etesync-server",
+    "displayName": "ETESync",
+    "description": "ETESync one click app for caprover",
```

Comment:
Can you please provide a better description? This is not a well known application. 
In a sentence or two, briefly explain what this is.

Reply:
https://github.com/caprover/one-click-apps/pull/139/commits/ea15ee4dcf3518aad39ba0662f08f32ac4207d46